Source/Core/VideoBackends/Vulkan/main.cpp

```diff
@@ -289,32 +277,29 @@ void VideoBackend::Shutdown()
 {
   g_command_buffer_mgr->WaitForGPUIdle();
 
+  g_perf_query.reset();
+  g_texture_cache.reset();
+  g_vertex_manager.reset();
+  g_renderer.reset();
+  StateTracker::DestroyInstance();
+  g_framebuffer_manager.reset();
+  g_shader_cache->Shutdown();
```

Comment:
If we make the above change, this will need to have an if (g_shader_cache) around it.

Reply:
Thanks for the heads up. I've also added a check for g_command_buffer_mgr.